content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Generate Istio Metrics Without Mixer
+description: In-proxy generation of service-level metrics.
+weight: 70
+---
+
+{{< warning >}}
+
+This document describes **Alpha** quality features. They have been exposed *only* to enable further testing
+and evaluation across the community.
+
+The functionality described here is NOT considered stable. No performance or stability guarantees are provided
+at this time.
+
+{{< /warning>}}
+
+Istio 1.3 adds experimental support to generate service-level HTTP metrics
+directly in the Envoy proxies. This feature lets you continue to monitor your service meshes using the tools Istio provides
+without needing Mixer. This support is the first step towards the new
```

Comment:
is there any stable link for the Mixer v2 plan / a high level doc?

Reply:
Not published to the website. I also don't believe we have a full up-to-date doc that we can point to here. Stay tuned, however.

Reply:
I've removed the references here. That detail isn't needed.